docs/core/preview3/migration/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: .NET Core Application Deployment | Microsoft Docs
+description: .NET Core project.json to csproj migration
+keywords: .NET, .NET Core, .NET Core migration
+author: blackdwarf
+ms.author: mairaw
+ms.date: 02/17/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 1feadf3d-3cfc-41dd-abb5-a4fc303a7b53
+---
+
+# Migrating .NET Core projects
+
+[!INCLUDE[preview-warning](../../../includes/warning.md)]
+
+This document will cover migration scenarios for .NET Core projects. It will cover the following three migration scenarios:
+
+1. Migration from a valid latest schema of `project.json` to `csproj`
+2. Migration from DNX to a valid Preview 2 project.json
+3. Migration from RC3 and previous .NET Core csproj projects to the final format 
+
+## Migration from project.json to csproj
+Migration from project.json to csproj can be done via the [`dotnet migrate`](../tools/(dotnet-migrate.md) command-line tool or through Visual Studio 2017. Both of these migrations use the same underlying engine to migrate projects, which means that the results will be the same regardless of the method you choose. In high majority of cases, using one of these two ways to migrate the project.json to csproj is the only thing that is needed and no further manual editing of the csproj is necessary. 
+
+Visual Studio 2017 will migrate the project automatically on opening either the `xproj` file or the solution file which references `xproj` files. If a solution is opened, Visual Studio 2017 will automatically migrate all of the projects that are specified in the solution. 
+
+In the command line scenario, you can use the `dotnet migrate` command. It will migrate a project, a solution or a set of folders in that order depending on which ones were found. The resulting csproj file will be named per the `name` property in project.json or, lacking that, per containing directory name. 
+
+In both migration modalities files that were migrated (project.json, global.json and xproj) will be moved to a `backup` folder. Any solution will also be upgraded to a new version number so that it cannot be accidentally opened in versions of Visual Studio earlier than 2017. 
```

Comment:
modalities -> options or modes? also add a comma after that

On Windows, the backup folder is with a capital B `Backup`.

Any solution will also be upgraded to a new version number so that it cannot be accidentally opened in versions of Visual Studio earlier than 2017.  -> The solution files will also be upgraded to Visual Studio 2017 and you won't be able to open that solution in previous versions of Visual Studio.